core/org.openjdk.jmc.flightrecorder/src/main/java/org/openjdk/jmc/flightrecorder/stacktrace/StacktraceModelUtils.java

```diff
@@ -0,0 +1,23 @@
+package org.openjdk.jmc.flightrecorder.stacktrace;
+
+import org.openjdk.jmc.flightrecorder.stacktrace.StacktraceModel.Branch;
+import org.openjdk.jmc.flightrecorder.stacktrace.StacktraceModel.Fork;
+
+/**
+ * StacktraceUtils useful shared functionality on {@link StacktraceModel}
+ */
+public final class StacktraceModelUtils {
```

Comment:
I.e. JMC-6787 is about moving out all (selection) state from the stack trace model and making it into two pieces - one stateless model that models the full tree, and another one which models the selection states in that model. They are currently one, since it's way more efficient that way. It may be the case that we should have two different models - the current one, and support selections and state in it, and another one which simply models the aggregated tree. That, however, is for another day.